apps/extension/src/utils/download-proving-keys.ts

```diff
@@ -3,8 +3,9 @@ import * as fs from 'fs';
 import * as path from 'path';
 import { provingKeys } from '@penumbra-zone/types/src/proving-keys';
 
+// TODO: When v0.65.0 launches, change this back to a git tag path
 const githubSourceDir =
-  'https://github.com/penumbra-zone/penumbra/raw/v0.64.1/crates/crypto/proof-params/src/gen/';
+  'https://github.com/penumbra-zone/penumbra/raw/main/crates/crypto/proof-params/src/gen/';
```

Comment:
Unfortunately, we have to temporarily revert this https://github.com/penumbra-zone/web/pull/351

Reply:
Make sure to remove your `/dist/` folder so these re-download